Makefile

```diff
@@ -44,6 +44,9 @@ help:
 	@echo "  test-foreman-smoke         to perform a generic smoke test"
 	@echo "  graph-entities             to graph entity relationships"
 	@echo "  lint                       to run pylint on the entire codebase"
+	@echo "  join-logs                  to join xdist log files into one"
```

Comment:
I would name all logs targets using the same prefix, for example logs-join and logs-clean.